src/units/context-report.js

```diff
@@ -14,10 +14,17 @@ unit.createContextMenuCommand()
 		const modChannel = unit.config.guildConfigs[interaction.guildId].modChannel;
 		const modRole = unit.config.guildConfigs[interaction.guildId].modRole;
 		const channel = interaction.client.channels.cache.get(modChannel);
+		const reportedMessage = interaction.targetMessage;
 		const embed = new GodotEmbedBuilder()
-			.setTitle('Reported Message from ' + interaction.user.username + ':')
-			.setDescription(`https://discord.com/channels/${interaction.guildId}/${interaction.channelId}/${interaction.targetId}`);
-		channel.send({ content: `<@&${modRole}>`, embeds: [embed] });
+			.setAuthor({ name: interaction.user.username, iconURL: interaction.user.avatarURL() ?? interaction.user.defaultAvatarURL })
+			.setDescription(reportedMessage.content || 'No content');
```

Comment:
Just an idea, not really needed: we could make the text smaller and grayed out with this syntax
```suggestion
			.setDescription(reportedMessage.content || '-# no content');
```